docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -1090,6 +1090,34 @@ file and both must be stored in the same file. Example usage:
   URI option values must be properly URI escaped. This applies, for example, to
   slashes in the paths.
 
+Modifying ``SSLContext``
+------------------------
+It may be desirable to further configure TLS options in the driver, for example
+by enabling or disabling certain ciphers. Currently, the Ruby driver does not
+provide a way to do this when initializing a ``Mongo::Client``.
+
+However, the Ruby driver provides a way to set global "TLS context hooks" --
+these are user-provided methods that will be invoked before any SSL socket
+connection and can be used to modify the underlying ``OpenSSL::SSL::SSLContext``
+object used by the socket.
+
+To set the TLS context hooks, use the ``Mongo.tls_context_hooks`` method, and
+provide an Array of ``Procs`` that will be run prior to connecting any SSL socket:
+
+.. code-block:: ruby
+
+  Mongo.tls_context_hooks = [
+    Proc.new { |context|
+      context.ciphers = ["AES256-SHA"]
+    }
+  ]
```

Comment:
Let me know if there's a more realistic example than this -- I'm not sure how people would actually want to use this.

Reply:
ALSO, let me know if we should recommend including this code in any specific part of a user's application?

Reply:
The use case requested in the ticket was removing a cipher, but I don't know how that works exactly. If you'd like to research that I would suggest giving that example, but the way this example is now is also fine with me.

Reply:
For the location, I can suggest:

> Set the TLS context hooks before creating any `Mongo::Client` instances. For example, in a Rails application this code could be placed in an initializer.

Reply:
I think the array assignment can easily become problematic and it should not be recommended. I suggest instead

```
  Mongo.tls_context_hooks.push(
    Proc.new { |context|
      context.ciphers = ["AES256-SHA"]
    }
  )
```